tensorflow/api_template_v1.__init__.py

```diff
@@ -40,7 +40,8 @@
   vars()['__all__'].append('contrib')
 
 from tensorflow.python.platform import flags  # pylint: disable=g-import-not-at-top
-app.flags = flags  # pylint: disable=undefined-variable
+from tensorflow.python.platform import app  # pylint: disable=g-import-not-at-top
```

Comment:
This change fixs cmake's built, it didn't execute `BUILD`

Reply:
This change broke direct imports of top-level modules, because the `app` module is being used below on line 48 to extract the directory of the generated API modules and add it to `tensorflow.__path__`.  Previously the `app` name referred to the generated API module, but this change makes it the version under `tensorflow.python` and as a result a different API directory gets added to `__path__`.

I have PR #24867 out to revert this change so we unbreak the import logic.  I would have tried to fix-forward but there isn't enough context in this PR to determine why this change was necessary to fix the cmake build.